### PR TITLE
fix: Switch from travis-ci.org to GitHub Actions CI and setup semantic-release

### LINKED ISSUE #47: Switch from travis-ci.org to GitHub Actions CI and setup semantic-release

### PR DESCRIPTION
**Pull request by Gabriel-403**
Fixes #47

### CODE REVIEW

**Comment by Zxilly** [comment]
plz squash your commit

**Review by Zxilly — APPROVED** [review]
plz squash commits

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,3 +37,18 @@ jobs:
...
+    needs:  build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run semantic-release
+        if: github.repository == 'node-casbin/sequelize-adapter' && github.event_name == 'push'
+        run: |
+          yarn install --no-lockfile
+          yarn run prepack
+          yarn run release
```
Comment (Zxilly) [inline .github/workflows/ci.yml:51]: `release` didn't exist. You should first install `semantic-release` or use `npx`.

**Review by Zxilly — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Gabriel-403) [reply .github/workflows/ci.yml:51]: thanks for help me find issue

**Comment by hsluoyz** [comment]
@Gabriel-403 plz fix:

image

**Comment by Gabriel-403** [comment]
> @Gabriel-403 plz fix:
> 
> image

working on this

File: package.json [hunk 2]
```diff
@@ -11,7 +11,10 @@
     "lint": "eslint --ext .ts src/ test/",
     "test": "jest --forceExit",
     "format": "yarn format:check --write",
-    "format:check": "prettier --check \"**/*.{md,json}\" \"{src,test}/**/*.ts\""
+    "format:check": "prettier --check \"**/*.{md,json}\" \"{src,test}/**/*.ts\"",
+    "prepack": "run-s build",
```
Comment (Zxilly) [inline package.json:17]: `run-s` is provided by package `npm-run-all`, you should add it to `devDependencies`

**Review by Zxilly — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@Zxilly do you have any advice?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -37,3 +37,18 @@ jobs:
         with:
           github-token: ${{ secrets.github_token }}
           parallel-finished: true
+
+  semantic-release:
+    needs:  build
```
Comment (Zxilly) [inline .github/workflows/ci.yml:42]: It will be better if you also add `finish` to needs

File: package.json [hunk 4]
```diff
@@ -1,101 +1,54 @@
 {
-  "name": "casbin-sequelize-adapter",
-  "version": "2.2.0",
-  "description": "Sequelize adapter for Casbin",
-  "main": "lib/adapter.js",
-  "typings": "lib/adapter.d.ts",
+  "name": "casbin.js",
```
Comment (Zxilly) [inline package.json:2]: well, why do you replace the `package.json`?

**Review by Zxilly — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
Plz hurry up. We need to close this ticket in one or two days, because there are another two PRs waiting for this PR to be merged.

**Comment by Gabriel-403** [comment]
> Plz hurry up. We need to close this ticket in one or two days, because there are another two PRs waiting for this PR to be merged.

Finish today

**Pull request closed without merging**